code/index.html

```diff
@@ -1,22 +1,146 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta charset="utf-8"
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Project 2: News-site Economics Today</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
   </head>
+  
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <!--***Header desktop***-->
+    <header class="header sticky-top">   
+       <nav class="menu">
+          <a href="https://www.technigo.io/">
+            <img class="logo" 
+            src="./logo.svg"
+            alt="Technigo-company-logo"/>  
+         </a>
+         <div class="site-name"><b>ECONOMICS TODAY</b></div>    
+         <ul class="menu-links"> 
+           <li class="link"><a href="#">ABOUT</a></li>
+           <li class="link"><a href="#">CONTACT</a></li>
+           <li class="link"><a href="#">READ MORE</a></li>
+           <li class="link"><a href="#">PRESS</a></li>
+         </ul>   
+      </nav>
+   </header>
+   <!--***Header tablet***-->
+   <header class="header-tablet sticky-top">   
+        <a class="item-link" href="#">ABOUT</a>
+        <a class="item-link" href="#">CONTACT</a>
+        <a class="item-logo" href="https://www.technigo.io/"><img class="logo-tablet" src="./logo.svg" alt="Technigo logo"/></a> 
+        <a class="item-link" href="#">READ MORE</a>
+        <a class="item-link" href="#">PRESS</a>        
     </header>
```

Comment:
Instead of creating the same content over and over, we can change the position and styling in the CSS file 🙌